### PR TITLE
add fileInfoDao and ut

### PR DESCRIPTION
**Pull request by chenyao1**

### CODE REVIEW

File: smart-common/src/main/java/org/smartdata/common/models/FileInfo.java [hunk 1]
```diff
@@ -165,6 +165,27 @@ public static FileInfo fromHdfsFileStatus(HdfsFileStatus status, String parent)
         .build();
   }
 
+  @Override
+  public boolean equals(Object o) {
```
Comment (littlezhou) [inline smart-common/src/main/java/org/smartdata/common/models/FileInfo.java]: equals(FileInfo) is enough, it does not to be Object

File: smart-common/src/main/java/org/smartdata/common/models/FileInfo.java [hunk 2]
```diff
@@ -165,6 +165,27 @@ public static FileInfo fromHdfsFileStatus(HdfsFileStatus status, String parent)
...
+    if (this == o) return true;
+    if (o == null || getClass() != o.getClass()) return false;
+
+    FileInfo fileInfo = (FileInfo) o;
+
+    if (fileId != fileInfo.fileId) return false;
```
Comment (littlezhou) [inline smart-common/src/main/java/org/smartdata/common/models/FileInfo.java]: combine some of the comparisons together,  and using {} even it contains only 1 statement

File: smart-metastore/src/test/java/org/smartdata/metastore/tables/TestFileInfoDao.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
+package org.smartdata.metastore.tables;
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/tables/TestFileInfoDao.java]: License statement

File: smart-metastore/src/test/java/org/smartdata/metastore/tables/TestFileInfoDao.java [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+    fileInfoDao.deleteById(fileId);
+    fileInfos = fileInfoDao.getAll();
+    Assert.assertTrue(fileInfos.size() == 1);
+    fileInfoDao.deleteAll();
+    fileInfos = fileInfoDao.getAll();
+    Assert.assertTrue(fileInfos.size() == 0);
+  }
+}
+
+
```
Comment (littlezhou) [inline smart-metastore/src/test/java/org/smartdata/metastore/tables/TestFileInfoDao.java]: remove blank lines

File: smart-common/src/main/java/org/smartdata/common/models/FileInfo.java (hunk 1 above)
Comment (chenyao1) [inline smart-common/src/main/java/org/smartdata/common/models/FileInfo.java]: solve

**Pull request merged**